app/controllers/ember-community-survey-2017.js

```diff
@@ -391,118 +341,88 @@ const longLived = {
         15.8, // '3–6 Months',
         19.8, // '< 3 Months',
       ],
-      color: emberOrange,
     },
   ],
-};
+}).highchartsOptions;
 
-const serverLanguage = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text: 'What is the primary language of your server-side stack?',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
+const serverLanguage = new VerticalBarChart({
+  chart: {
+    categories: [
+      'Ruby',
+      'JavaScript',
+      'Java',
+      'Python',
+      'PHP',
+      '.NET',
+      'Elixir / Erlang',
+      'Go',
+      'Scala',
+      'TypeScript',
+      'C++',
+      'Other',
+    ],
+    title: 'What is the primary language of your server-side stack?',
   },
-  data: [
+
+  rawData: [
     {
-      name: '2016',
       color: darkGrayColor,
-      data: [
-        { y: 41, name: 'Ruby' },
-        { y: 24, name: 'JavaScript' },
-        { y: 17, name: 'Java' },
-        { y: 13, name: 'PHP' },
-        { y: 12, name: 'Python' },
-        { y: 8, name: '.NET' },
-        { y: 6, name: 'Elixir / Erlang' },
-        { y: 5, name: 'Go' },
-        { y: 2, name: 'Scala' },
-        { y: 0, name: 'TypeScript' },
-        { y: 0, name: 'C++' },
-        { y: 1, name: 'Rust' },
-        { y: 3, name: 'Other' },
-      ],
+      label: '2016',
+      values: [41, 24, 17, 13, 12, 8, 6, 5, 2, 0, 0, 1, 3],
     },
     {
-      name: '2017',
       color: emberOrange,
-      data: [
-        { name: 'Ruby', y: 36.97 },
-        { name: 'JavaScript', y: 24.54 },
-        { name: 'Java', y: 20.57 },
-        { name: 'Python', y: 13.5 },
-        { name: 'PHP', y: 12.43 },
-        { name: '.NET', y: 8.71 },
-        { name: 'Elixir / Erlang', y: 7.95 },
-        { name: 'Go', y: 6.44 },
-        { name: 'Scala', y: 2.65 },
-        { name: 'TypeScript', y: 1.58 },
-        { name: 'C++', y: 1.2 },
-        { name: 'Other', y: 2.97 },
+      label: '2017',
+      values: [
+        36.97,
+        24.54,
+        20.57,
+        13.5,
+        12.43,
+        8.71,
+        7.95,
+        6.44,
+        2.65,
+        1.58,
+        1.2,
+        2.97,
```

Comment:
Before the changes, the `data` arrays for 2016 and 2017 didn't list the categories in the same order (e.g. PHP and Python). 2017 is also missing `Rust`.

As a result, we unfortunately want to be careful when creating tuples. Luckily, we have Percy to tell us if we might have switched an order.

<img width="700" alt="percy snapshot" src="https://user-images.githubusercontent.com/16869656/112098446-16e4fa80-8ba2-11eb-976d-103c6cb2c577.png">

```suggestion
      values: [
        36.97,
        24.54,
        20.57,
        12.43,
        13.5,
        8.71,
        7.95,
        6.44,
        2.65,
        1.58,
        1.2,
        0,
        2.97,
```